teamsyncd/teamsync.cpp

```diff
@@ -121,6 +131,19 @@ TeamSync::TeamPortSync::TeamPortSync(const string &lagName, int ifindex,
                            "Unable to register port change event");
     }
 
+    m_tdc = teamdctl_alloc();
+
+    if (m_tdc == NULL)
+    {
+        SWSS_LOG_THROW("Failed to allocate teamdctl context.");
+    }
+
+    err = teamdctl_connect(m_tdc, m_lagName.c_str(), NULL, NULL); 
```

Comment:
Where is this defined? I'm sorry but I couldn't find the function. Can you please tell me which file has it?